holoviews/plotting/plot.py

```diff
@@ -426,7 +426,7 @@ def __init__(self, element, keys=None, ranges=None, dimensions=None,
         keys = keys if keys else list(self.hmap.data.keys())
         plot_opts = self.lookup_options(self.hmap.last, 'plot').options
 
-        dynamic = element.mode if isinstance(element, DynamicMap) else False
+        dynamic = False if not isinstance(element, DynamicMap) or element.sampled else element.mode
```

Comment:
I think dynamic should be ``False`` or a string. i.e if sampled I think ``dynamic='sampled'``.